mimic/test/test_auth.py

```diff
@@ -14,6 +14,7 @@
 
 
 class ExampleCatalogEndpoint(object):
+
```

Comment:
Please fix your autopep8 to not introduce this blank-line noise :).

(Also, since you apparently did do this already, can you share how you configured it?)